manual/src/refman/extensions/tail_mod_cons.etex

```diff
@@ -0,0 +1,548 @@
+(Introduced in OCaml 4.14)
+
+Note: this feature is considered experimental, and its interface may
+evolve, with user feedback, in the next few releases of the language.
+
+Consider the simple way of implementing the "List.map" function:
+\begin{verbatim}
+let rec map f li =
+  match li with
+  | [] -> []
+  | x :: xs ->
+    let y = f x in
+    y :: map f xs
+\end{verbatim}
+
+A well-known limitation of this implementation is that the recursive
+call, "map f xs", is not in \emph{tail} position. The runtime needs to
+remember to continue with "y :: _" after the call returns and, as
+a result, this function consumes some amount of call-stack space on
```

Comment:
```suggestion
remember to continue with "y :: _" after the call returns and
this function consequently consumes some amount of call-stack space on
```
(I personally wouldn't use a subordinate clause here - the problem is a direct consequence)